hypothesis-python/src/hypothesis/statistics.py

```diff
@@ -62,6 +62,8 @@ def __init__(self, engine):
             self.exit_reason = 'nothing left to do'
         elif engine.exit_reason == ExitReason.flaky:
             self.exit_reason = 'test was flaky'
+        elif engine.exit_reason == ExitReason.max_shrinks:  # pragma: no cover
```

Comment:
Why can’t we test this?

Reply:
:man_facepalming: in typing up the explanation, I have of course seen how to test this.  One moment...

Reply:
😄